CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 13.0.0
+
+#### Breaking changes
+
+* Text out of tags is no longer allowed in XML parameters. This prevents to add "comments" that would be lost by automated transformation of these parameters.
+
+#### New features
+
+* The attribute "reference" can be used in XML parameters to document a source.
+
```

Comment:
In general, prefer the imperative mode in Changelog :

* Disallow text out of tags in XML parameters
    * This prevents...

- Introduce a "reference" attribute to document a source in XML parameters